frontend/src/components/phones/dashboard/Dashboard.tsx

```diff
@@ -1,12 +1,180 @@
+          }`}
+        >
+          <ForwardIcon
+            alt="Forwarding All Messages"
+            className={styles["forward-icon"]}
+            width={15}
+            height={15}
+          />
+        </button>
+        <button
+          onClick={toggleForwarding}
+          className={`${styles["base-button"]} ${
+            enableForwarding ? "" : styles["active-button"]
+          }`}
+        >
+          <BlockIcon
+            alt="Blocking All Messages"
+            className={styles["block-icon"]}
+            width={15}
+            height={15}
+          />
+        </button>
+      </div>
+      <div className={styles["phone-controls-description"]}>
+        {enableForwarding ? (
+          <span>
+            Relay is currently forwarding all phone calls and SMS messages to
+            your true phone number.
+          </span>
+        ) : (
+          <span>
+            Relay is blocking all phone calls and text messages—you will not
+            receive anything from your phone number mask.
+          </span>
+        )}
+      </div>
+    </div>
+  );
+
+  const phoneMetadata = (
+    <div className={styles["metadata-container"]}>
+      <dl>
+        <div className={`${styles["forward-target"]} ${styles.metadata}`}>
+          <dt>Forwarded to:</dt>
+          <dd>Add user phone number here</dd>
+        </div>
+        <div className={`${styles["date-created"]} ${styles.metadata}`}>
+          <dt>Date Created:</dt>
+          <dd>Date</dd>
+        </div>
+      </dl>
+    </div>
+  );
 
   return (
     <main>
-      Phone dashboard comes here! Your Relay phone mask is{" "}
-      <samp>{relayNumberData.data?.[0]?.number ?? "still loading"}</samp>.
+      <div className={styles["dashboard-card"]}>
+        <span className={styles["header-phone-number"]}>
+          {e164PhoneNumberFormat(relayNumberData.data?.[0].number!)}
+          <span className={styles["copy-controls"]}>
+            <span className={styles["copy-button-wrapper"]}>
+              <button
+                type="button"
+                className={styles["copy-button"]}
+                title="Copied"
+                onClick={copyPhoneNumber}
+              >
+                <CopyIcon
+                  alt="test"
+                  className={styles["copy-icon"]}
+                  width={32}
+                  height={32}
+                />
+              </button>
+              <span
+                aria-hidden={!justCopiedPhoneNumber}
+                className={`${styles["copied-confirmation"]} ${
+                  justCopiedPhoneNumber ? styles["is-shown"] : ""
+                }`}
+              >
+                Copied!
+              </span>
+            </span>
+          </span>
+        </span>
+
+        {phoneStatistics}
+        {phoneControls}
+        {phoneMetadata}
+      </div>
     </main>
   );
 };
+
+function e164PhoneNumberFormat(relayNumber: string) {
```

Comment:
suggestion (non-blocking):
```suggestion
function formatPhoneNumberToUSDisplay(e164Number: string) {
```

Reply:
Done!